tests/pyfunc/test_spark.py

```diff
@@ -1361,7 +1361,7 @@ def predict(self, context, model_input):
     udf = mlflow.pyfunc.spark_udf(spark=spark, model_uri=save_path, result_type="string")
     pdf = df.withColumn("output", udf("str", "arr", "obj", "obj_arr")).toPandas()
     assert pdf["output"][0] == "a | [0] | {'bool': True} | [{'double': 0.1}]"
-    assert pdf["output"][1] == "b | [1, 2] | {'bool': False} | [{'double': 0.2}, {'double': 0.3}]"
```

Comment:
Note: Those input lists like `[1, 2]` are converted to numpy array when wrapped by pandas DataFrame. Back then this test was added, `_enforce_array` returned list, but now it returns same numpy array. This cause slight change in how it rendered as string here.